src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -234,6 +235,7 @@ protected OptionParser buildOptionParser() {
 		parser.accepts( "rabbitmq_exchange_durable", "Exchange durability. Default is disabled" ).withOptionalArg();
 		parser.accepts( "rabbitmq_exchange_autodelete", "If set, the exchange is deleted when all queues have finished using it. Defaults to false" ).withOptionalArg();
 		parser.accepts( "rabbitmq_routing_key_template", "A string template for the routing key, '%db%' and '%table%' will be substituted. Default is '%db%.%table%'." ).withRequiredArg();
+		parser.accepts( "rabbitmq_message_persistent", "Message persistence. Defaults to false" ).withOptionalArg();
```

Comment:
here should be `withRequiredArg();`, meaning if you provide `--rabbitmq_message_persistent` option, you need to have a value for this option.

Reply:
Actually this is OK, we do usually use withOptionalArg for booleans